test/gui/shared/steps/steps.py

```diff
@@ -399,14 +399,57 @@ def createFolder(context, foldername, username=None):
     path = join(syncPath, foldername)
     os.makedirs(path)
 
-
+ 
+@When('user "|any|" creates "|any|" files each of size "|any|" bytes inside the folder "|any|"')
+def step(context, username, filenumber, filesize, foldername):
+    syncPath = context.userData['currentUserSyncPath']
+    path = join(syncPath, foldername)
+    filesize = builtins.int(filesize)
+    # Syncing all the created files at once seems to have some problem on syncing
+    # So once a file is synced the process have to wait unless the next file is created
+    # So we added Snooze of 5 sec
```

Comment:
```suggestion
    # A file is scheduled to be synced but is marked as ignored for 5 seconds. And if we try to sync it, it will fail. So we need to wait for 5 seconds.
    # https://github.com/owncloud/client/issues/9325
```